api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -161,7 +169,7 @@ def publish_cdr(cmd_name, args)
     common.run_inline %W{gcloud auth activate-service-account -q --key-file #{key_file.path}}
 
     source_cdr_project = op.opts.source_cdr_project_override || tier.fetch(:source_cdr_project)
-    source_dataset = "#{tier.fetch(:source_cdr_project)}:#{op.opts.bq_dataset}"
+    source_dataset = "#{source_cdr_project}:#{op.opts.bq_dataset}"
```

Comment:
this fixes a bug where the source_dataset wasn't being overrode by the input argument